### PR TITLE
[BUGFIX/PATCH]: Markdown Documentation Cleanup

### PR DESCRIPTION
**Pull request by JVNpixels**
Fixes the README markdown file and gives proper credit to those who contributed to the development of Friday Night Funkin’.

### CODE REVIEW

**Comment by JVNpixels** [comment]
I know there was In-Game credits, but atleast update the GitHub README.

**Comment by JVNpixels** [comment]
These were to just to start, and maybe you guys could do the rest? I mean you guys know their socials and I dont.

**Comment by JVNpixels** [comment]
Just to clarify, what needs revision? I was writing this when I was tired and I also just woke up.

**Comment by JVNpixels** [comment]
Oh I got it now, thanks, I will update this later.

**Comment by JVNpixels** [comment]
I have added everyone from the development team and contributing help (like sr pelo and tankman), note I only added the developers and helpers, not the Kickstarter or legal stuff, that can be seen with-in the games credits menu.

**Comment by EliteMasterEric** [comment]
@ninjamuffin99 Does this look good? Any thoughts/additions?

File: README.md [hunk 1]
```diff
@@ -19,26 +20,48 @@ Please check out our [Contributor's guide](./CONTRIBUTORS.md) on how you can act
...
+- [IvanAlmighty](https://twitter.com/IvanA1mighty) - Misc UI Design, Voice of Darnell
 
 ## Music
-- [Kawaisprite](https://twitter.com/kawaisprite) - Musician
+- [Kawaisprite](https://twitter.com/kawaisprite) - Lead Musician
+- [Saruky](https://twitter.com/Saruky__) - Musician
+- [SixImpala](https://twitter.com/SixImpala) - Musician
+- [Saster](https://twitter,com/sub0ru) - Featured Guest Musician for "Thorns Erect"
+- [nuphory](https://twitter.com/nuphory) - Featured Guest Musician for "Results (Excellent)
+- [MtH](https://twitter.com/emmnyaa) - Featured Guest Musician for "Breakfast (Week 6 Version)
```
Comment (Hundrec) [inline README.md:59]: Make sure to add end quotes after Results and Breakfast!

**Review by Hundrec — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Hundrec) [reply README.md]: Remember to add quotes to Results (Excellent) too :)

**Review by Hundrec — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -19,26 +20,48 @@ Please check out our [Contributor's guide](./CONTRIBUTORS.md) on how you can act
...
+- [PennilessRagamuffin](https://twitter.com/PennilessRagamuffin) - Cutscene Animation
 - [Moawling](https://twitter.com/moawko) - Week 6 Pixel Art
-- [IvanAlmighty](https://twitter.com/IvanA1mighty) - Misc UI Design
+- [IvanAlmighty](https://twitter.com/IvanA1mighty) - Misc UI Design, Voice of Darnell
 
 ## Music
-- [Kawaisprite](https://twitter.com/kawaisprite) - Musician
+- [Kawaisprite](https://twitter.com/kawaisprite) - Lead Musician
+- [Saruky](https://twitter.com/Saruky__) - Musician
+- [SixImpala](https://twitter.com/SixImpala) - Musician
```
Comment (doggogit) [inline README.md:56]: Handle should be changed to:
```suggestion
- [SixImpala](https://twitter.com/siximpalasix) - Musician
```

File: README.md [hunk 3]
```diff
@@ -19,26 +20,48 @@ Please check out our [Contributor's guide](./CONTRIBUTORS.md) on how you can act
...
 ## Art / Animation / UI
 - [PhantomArcade3K](https://twitter.com/phantomarcade3k) - Artist and Animator
 - [Evilsk8r](https://twitter.com/evilsk8r) - Art
+- [beck](https://twitter.com/beck) - Art, Cutscene Cleanup, Cutscene Background Art
+- [Red Minus](https://twitter.com/RedMinus) - Additional Background Design
+- [Figburn](https://twitter.com/Figburn) - Cutscene Animation
+- [Sade](https://twitter.com/Sade) - Cutscene Animation
+- [Topium](https://twitter.com/Topium) - Cutscene Animation
+- [BlairTheUnseriousGuy](https://twitter.com/BlairTheUnseriousGuy) - Cutscene Animation
+- [PennilessRagamuffin](https://twitter.com/PennilessRagamuffin) - Cutscene Animation
```
Comment (doggogit) [inline README.md:49]: Same goes here, accounts are either incorrect or just straight up don't exist.

File: README.md [hunk 4]
```diff
@@ -19,26 +20,48 @@ Please check out our [Contributor's guide](./CONTRIBUTORS.md) on how you can act
...
 - [EliteMasterEric](https://twitter.com/EliteMasterEric) - Programmer
 - [MtH](https://twitter.com/emmnyaa) - Charting and Additional Programming
+- [Jenny Crowe](https://twitter.com/JennyCrowe) - Additional Programming
+- [hazel](https://twitter.com/hazel) - Additional Programming, Devops and Additional Internal Tooling
+- [Mike Welsh](https://twitter.com/MikeWelsh) - Additional Programming
+- [Saharan](https://twitter.com/Saharan) - Additional Programming
+- [Ian Harrigan](https://twitter.com/IanHarrigan) - Additional Programming
+- [Osaka Red LLC: Thomas J Webb](https://twitter.com/ThomasJWebb) - Additional Programming
 - [GeoKureli](https://twitter.com/Geokureli/) - Additional Programming
+- [Will Blanton](https://twitter.com/WillBlanton) - Additional Programming
```
Comment (doggogit) [inline README.md:36]: Abovementioned comment also applies to this user.

File: README.md [hunk 5]
```diff
@@ -19,26 +20,48 @@ Please check out our [Contributor's guide](./CONTRIBUTORS.md) on how you can act
...
 ## Programming
 - [ninjamuffin99](https://twitter.com/ninja_muffin99) - Lead Programmer
 - [EliteMasterEric](https://twitter.com/EliteMasterEric) - Programmer
 - [MtH](https://twitter.com/emmnyaa) - Charting and Additional Programming
+- [Jenny Crowe](https://twitter.com/JennyCrowe) - Additional Programming
+- [hazel](https://twitter.com/hazel) - Additional Programming, Devops and Additional Internal Tooling
+- [Mike Welsh](https://twitter.com/MikeWelsh) - Additional Programming
+- [Saharan](https://twitter.com/Saharan) - Additional Programming
+- [Ian Harrigan](https://twitter.com/IanHarrigan) - Additional Programming
+- [Osaka Red LLC: Thomas J Webb](https://twitter.com/ThomasJWebb) - Additional Programming
```
Comment (doggogit) [inline README.md:34]: I believe that the aforementioned people here have been attached to incorrect Twitter accounts or accounts that simply don't exist. 

Maybe someone from the team should chime in and examine and see if they can correct the mistakes and say who does or doesn't have Twitter and/or any other social media platform.

File: README.md [hunk 6]
```diff
@@ -19,26 +20,48 @@ Please check out our [Contributor's guide](./CONTRIBUTORS.md) on how you can act
 
 # Credits and Special Thanks
 
-Full credits can be found in-game, or wherever the credits.json file is.
+The full credits can be found in-game, or wherever the credits.json file is.
```
Comment (doggogit) [inline README.md:23]: `The full` can be reverted back to `Full`, and here's the link to `credits.json` as follows:
```suggestion
Full credits can be found in-game, or [in the `credits.json` file](https://github.com/FunkinCrew/funkin.assets/blob/main/exclude/data/credits.json).
```

**Review by doggogit — CHANGES_REQUESTED** [review]
Some adjustments can be made, as shown here:

**Comment by doggogit** [comment]
> Some adjustments can be made, as shown here:

In addition, maybe move this to its own separate file, like `CREDITS.md`?

**Comment by JVNpixels** [comment]
> > Some adjustments can be made, as shown here:
> 
> In addition, maybe move this to its own separate file, like `CREDITS.md`?
> 
> Reason being that it might clutter the README.

Alright I will fix this up and make it look nicer, and then make a credits.md file

**Comment by doggogit** [comment]
> > > Some adjustments can be made, as shown here:
> > 
> > 
> > In addition, maybe move this to its own separate file, like `CREDITS.md`?
> > Reason being that it might clutter the README.
> 
> Alright I will fix this up and make it look nicer, and then make a credits.md file

Sweet! Just make sure to put the CREDITS.md in `docs/` whenever you end up making it.

@EliteMasterEric or @ninjamuffin99, can you guys confirm or provide some of the Twitter links for specific members that were credited/thanked?

**Comment by JVNpixels** [comment]
Note: I have found links for the following:
Sade
BlairTheUnseriousGuy
Topium
Ragamuffin

**Comment by doggogit** [comment]
> Note: I have found links for the following: Sade BlairTheUnseriousGuy Topium Ragamuffin

Awesome, I know hazel doesn't use twitter, they use mastodon... but I don't quite the remember the link..

**Comment by JVNpixels** [comment]
Credits will be up in a sec, I am quite busy rn

**Comment by doggogit** [comment]
> Credits will be up in a sec, I am quite busy rn

That's fine, take yer time

**Comment by JVNpixels** [comment]
Should be added now!

File: README.md [hunk 7]
```diff
@@ -19,26 +26,4 @@ Please check out our [Contributor's guide](./CONTRIBUTORS.md) on how you can act
...
-
-## Music
-- [Kawaisprite](https://twitter.com/kawaisprite) - Musician
-- [BassetFilms](https://twitter.com/Bassetfilms) - Music for "Monster", Additional Character Design
-
-## Special Thanks
-- [Tom Fulp](https://twitter.com/tomfulp) - For being a great guy and for Newgrounds
-- [JohnnyUtah](https://twitter.com/JohnnyUtahNG/) - Voice of Tankman
-- [L0Litsmonica](https://twitter.com/L0Litsmonica) - Voice of Mommy Mearest
+Full credits can be found in-game, [CREDITS.md,](https://github.com/FunkinCrew/Funkin) or [in the `credits.json` file](https://github.com/FunkinCrew/funkin.assets/blob/main/exclude/data/credits.json).
```
Comment (doggogit) [inline README.md:29]:
```suggestion
Full credits can be found in-game or in [CREDITS.md](https://github.com/FunkinCrew/Funkin).
```

File: docs/CREDITS.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
+# Credits and Special Thanks
+Full credits can be found in-game, or wherever the credits.json file is.
```
Comment (doggogit) [inline docs/CREDITS.md:2]:
```suggestion

```

**Review by doggogit — CHANGES_REQUESTED** [review]
Some stuff, idk, didn't like the credits.json file that much now lol

**Comment by doggogit** [comment]
You forgot to edit the lines in the README, but other than that, this is a pretty solid PR

**Review by doggogit — APPROVED** [review]
Once the Twitter links for specific members get sorted out, I've got no problem having this PR merged!

File: README.md (hunk 7 above)
Comment (Hundrec) [inline README.md:29]:
```suggestion
Full credits can be found in-game or in the [CREDITS.md](/docs/CREDITS.md) file.
```

**Comment by JVNpixels** [comment]
> You forgot to edit the line in the README, but other than that, this is a pretty solid PR

Which line was it again? I think it's just fine.

**Comment by doggogit** [comment]
> > You forgot to edit the line in the README, but other than that, this is a pretty solid PR
> 
> Which line was it again? I think it's just fine.

The one that you just fixed. It's good now!

**Comment by JVNpixels** [comment]
:D

**Comment by Hundrec** [comment]
I'd like to note that CREDITS.md appears to credit fewer people than credits.json does, so maybe we should keep credits.json easily accessible in the README

**Comment by JVNpixels** [comment]
@Hundrec, yes that would be a good idea.

**Comment by doggogit** [comment]
> I'd like to note that CREDITS.md appears to credit fewer people than credits.json does, so maybe we should keep credits.json easily accessible in the README     

Oh, didn't realize that, then sure, you can just put the line back in

File: README.md [hunk 9]
```diff
@@ -19,26 +26,4 @@ Please check out our [Contributor's guide](./CONTRIBUTORS.md) on how you can act
...
-
-## Music
-- [Kawaisprite](https://twitter.com/kawaisprite) - Musician
-- [BassetFilms](https://twitter.com/Bassetfilms) - Music for "Monster", Additional Character Design
-
-## Special Thanks
-- [Tom Fulp](https://twitter.com/tomfulp) - For being a great guy and for Newgrounds
-- [JohnnyUtah](https://twitter.com/JohnnyUtahNG/) - Voice of Tankman
-- [L0Litsmonica](https://twitter.com/L0Litsmonica) - Voice of Mommy Mearest
+Full credits can be found in-game or in the [credits.json](https://github.com/FunkinCrew/funkin.assets/blob/main/exclude/data/credits.json) file or in the [CREDITS.md](/docs/CREDITS.md) file.
```
Comment (doggogit) [inline README.md:29]:
```suggestion
Simplified credits can be found in [CREDITS.md](/docs/CREDITS.md), while full credits can be found in-game or [in the `credits.json` file](https://github.com/FunkinCrew/funkin.assets/blob/main/exclude/data/credits.json).
```

**Review by doggogit — CHANGES_REQUESTED** [review]
Like this?

File: README.md (hunk 9 above)
Comment (JVNpixels) [reply README.md:29]: Yes thank you!

**Review by doggogit — APPROVED** [review]

File: docs/CREDITS.md [hunk 10]
```diff
@@ -0,0 +1,45 @@
+# Credits and Special Thanks
+
+## Programming
+- [ninjamuffin99](https://twitter.com/ninja_muffin99) - Lead Programmer
+- [EliteMasterEric](https://twitter.com/EliteMasterEric) - Programmer
```
Comment (Hundrec) [inline docs/CREDITS.md:5]:
```suggestion
- [EliteMasterEric](https://twitter.com/EliteMasterEric) - Programmer
- [fabs](https://twitter.com/fabsthefabs) - Programmer
```

File: docs/CREDITS.md [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+- [MtH](https://twitter.com/emmnyaa) - Charting and Additional Programming
+- [Jenny Crowe](https://twitter.com/JennyCrowe) - Additional Programming
```
Comment (Hundrec) [inline docs/CREDITS.md:7]:
```suggestion
- [Jenny Crowe](https://twitter.com/jennyminer_biz) - Additional Programming
```

File: docs/CREDITS.md [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+- [hazel](https://twitter.com/hazel) - Additional Programming, Devops and Additional Internal Tooling
+- [Mike Welsh](https://twitter.com/MikeWelsh) - Additional Programming
+- [Saharan](https://twitter.com/Saharan) - Additional Programming
+- [Ian Harrigan](https://twitter.com/IanHarrigan) - Additional Programming
```
Comment (Hundrec) [inline docs/CREDITS.md:11]:
```suggestion
- [Ian Harrigan](https://twitter.com/ianharrigan1982) - Additional Programming
```

File: docs/CREDITS.md [hunk 13]
```diff
@@ -0,0 +1,45 @@
...
+- [Osaka Red LLC: Thomas J Webb](https://twitter.com/ThomasJWebb) - Additional Programming
+- [GeoKureli](https://twitter.com/Geokureli/) - Additional Programming
+- [Will Blanton](https://twitter.com/WillBlanton) - Additional Programming
```
Comment (Hundrec) [inline docs/CREDITS.md:14]:
```suggestion
- [Will Blanton](https://twitter.com/x01010111) - Additional Programming
```

File: docs/CREDITS.md [hunk 14]
```diff
@@ -0,0 +1,45 @@
...
+- [Cheemsandfriends](https://twitter.com/Cheemsandfriends) - FlxAnimate
```
Comment (Hundrec) [inline docs/CREDITS.md:15]:
```suggestion
- [Cheemsandfriends](https://twitter.com/CheemsnFriendos) - FlxAnimate
```

**Review by Hundrec — CHANGES_REQUESTED** [review]
I've compiled some Twitter handles (and personal websites if no Twitter) that I'm _fairly_ confident belong to the respective individuals

This is only for the Programming section, but I will add more as I find them!

File: docs/CREDITS.md [hunk 15]
```diff
@@ -0,0 +1,46 @@
...
+- Our contributors on GitHub
+
+## Art / Animation / UI
+- [PhantomArcade3K](https://twitter.com/phantomarcade3k) - Artist and Animator
+- [Evilsk8r](https://twitter.com/evilsk8r) - Art
+- [beck](https://twitter.com/beck) - Art, Cutscene Cleanup, Cutscene Background Art
+- [Red Minus](https://twitter.com/RedMinus) - Additional Background Design
+- [Figburn](https://twitter.com/Figburn) - Cutscene Animation
+- [Sade](https://twitter.com/Sade) - Cutscene Animation
```
Comment (Hundrec) [inline docs/CREDITS.md:25]:
```suggestion
- [Sade](https://twitter.com/sadewoo0) - Cutscene Animation
```

**Comment by JVNpixels** [comment]
@Hundrec, these changes should be added! Maybe next time I should think through thoroughly with things like these next time! 😅😅😅

File: docs/CREDITS.md [hunk 16]
```diff
@@ -0,0 +1,46 @@
...
+- [Topium](https://twitter.com/Topium) - Cutscene Animation
```
Comment (Hundrec) [inline docs/CREDITS.md:26]:
```suggestion
- [Topium](https://twitter.com/topium_) - Cutscene Animation
```

File: docs/CREDITS.md [hunk 17]
```diff
@@ -0,0 +1,46 @@
...
+- [BlairTheUnseriousGuy](https://twitter.com/BlairTheUnseriousGuy) - Cutscene Animation
+- [PennilessRagamuffin](https://twitter.com/PennilessRagamuffin) - Cutscene Animation
```
Comment (Hundrec) [inline docs/CREDITS.md:28]:
```suggestion
- [PennilessRagamuffin](https://twitter.com/matchragamuffin) - Cutscene Animation
```

File: docs/CREDITS.md [hunk 18]
```diff
@@ -0,0 +1,46 @@
...
+
+## Art / Animation / UI
+- [PhantomArcade3K](https://twitter.com/phantomarcade3k) - Artist and Animator
+- [Evilsk8r](https://twitter.com/evilsk8r) - Art
+- [beck](https://twitter.com/beck) - Art, Cutscene Cleanup, Cutscene Background Art
+- [Red Minus](https://twitter.com/RedMinus) - Additional Background Design
+- [Figburn](https://twitter.com/Figburn) - Cutscene Animation
+- [Sade](https://twitter.com/Sade) - Cutscene Animation
+- [Topium](https://twitter.com/Topium) - Cutscene Animation
+- [BlairTheUnseriousGuy](https://twitter.com/BlairTheUnseriousGuy) - Cutscene Animation
```
Comment (Hundrec) [inline docs/CREDITS.md:27]:
```suggestion
- [BlairTheUnseriousGuy](https://twitter.com/unseriousguy) - Cutscene Animation
```

File: docs/CREDITS.md [hunk 19]
```diff
@@ -0,0 +1,46 @@
...
+- [Osaka Red LLC: Thomas J Webb](https://haxe.social/@tjw) - Additional Programming
+- [GeoKureli](https://twitter.com/Geokureli/) - Additional Programming
+- [Will Blanton](https://twitter.com/WillBlanton) - Additional Programming
+- [Cheemsandfriends](https://twitter.com/Cheemsandfriends) - FlxAnimate
+- Our contributors on GitHub
+
+## Art / Animation / UI
+- [PhantomArcade3K](https://twitter.com/phantomarcade3k) - Artist and Animator
+- [Evilsk8r](https://twitter.com/evilsk8r) - Art
+- [beck](https://twitter.com/beck) - Art, Cutscene Cleanup, Cutscene Background Art
```
Comment (Hundrec) [inline docs/CREDITS.md:22]:
```suggestion
- [beck](https://twitter.com/?????) - Art, Cutscene Cleanup, Cutscene Background Art, Merch Design
```

I've got nothing for beck.

File: docs/CREDITS.md [hunk 20]
```diff
@@ -0,0 +1,46 @@
...
+- [Saster](https://twitter,com/sub0ru) - Featured Guest Musician for "Thorns Erect"
+- [nuphory](https://twitter.com/nuphory) - Featured Guest Musician for "Results (Excellent)"
+- [MtH](https://twitter.com/emmnyaa) - Featured Guest Musician for "Breakfast (Week 6 Version)"
+- [Kohta Takahashi](https://twitter.com/kohta09) - Featured Guest Musician for "Fresh Erect", "High Erect"
+- [BassetFilms](https://twitter.com/Bassetfilms) - Music for "Monster", Additional Character Design
+
+## Special Thanks
+- [Tom Fulp](https://twitter.com/tomfulp) - For being a great guy and for Newgrounds
+- [JohnnyUtah](https://twitter.com/JohnnyUtahNG/) - Voice of Tankman
+- [SrPelo](https://twitter.com/SrPelo) - Voice of Spooky Kids
```
Comment (Hundrec) [inline docs/CREDITS.md:45]:
```suggestion
- [SrPelo](https://twitter.com/_srpelo_) - Voice of Spooky Kids
```

**Review by Hundrec — CHANGES_REQUESTED** [review]
Here's another batch!

I think the rest of the links are fine, or are impossible to find without dev confirmation (beck)

**Comment by Hundrec** [comment]
> @Hundrec, these changes should be added! Maybe next time I should think through thoroughly with things like these next time! 😅😅😅

Don't worry about it, I'm glad you took the initiative to update the credits and make them more accessible!

**Comment by JVNpixels** [comment]
Alright, everything should be just fine now! 👍

**Comment by JVNpixels** [comment]
@Hundrec, I think this is good for approval, could you please review it?

**Comment by JVNpixels** [comment]
I’m pretty sure everything should be good now, and next time I do something documentation related (which I probably wont), then I will organize it and clean it up better as well as making ideas like to split the credits into its own file and stuff like that.

**Comment by Hundrec** [comment]
I think the structure and links look good now!

I'll approve this PR once we get official confirmation for beck's social media and verification for everyone else's links.

**Comment by JVNpixels** [comment]
> I think the structure and links look good now!
> 
> I'll approve this PR once we get official confirmation for beck's social media and verification for everyone else's links.

I found beck’s social media! Although their account is mostly for adult audiences, but I looked at the official FNF developers twitter page and who they were following and I found beck.

**Comment by Hundrec** [comment]
Nice sleuthing! Now we just need the devs' confirmation :)

**Comment by MAZ12211** [comment]
Just a suggestion, you might want to credit charters because fabs (along with spazkid) also charted a lot of erect remixes.

**Comment by ninjamuffin99** [comment]
I think we don't particularly need another credits related file to maintain and update. The JSON creds file is generally readable on web, and I don't think we particularly need links to socials for everyone in the credits, as it may not be their preference. If the people we work with ask for us to credit social medias, we will comply! 

So do appreciate the effort but I will close this PR!

**Comment by JVNpixels** [comment]
its ok it was like 2 months ago and hundrec did most of the commits

**Pull request closed without merging**